templates/map.html

```diff
@@ -205,6 +205,10 @@ <h3>Gym Sidebar</h3>
                 </div>
               </div>
                 {% endif %}
+              <div class="form-control" id="gym-name-filter-wrapper" style="display:none">
+                <h3>Name Includes</h3>
+                <input id="gyms-name-filter" type="text"></input>
```

Comment:
Can you change the id to `gym-name-filter` to make it consistent with the wrapper name and pokestop name filter id.